doc/python/subplots.md

```diff
@@ -354,7 +355,7 @@ fig.show()
 ```
 
 #### Subplots Types
-By default, the `make_subplots` function assumes that the traces that will be added to all subplots are 2-dimensional cartesian traces (e.g. `scatter`, `bar`, `histogram`, `violin`, etc.).  Traces with other subplot types (e.g. `scatterpolar`, `scattergeo`, `parcoords`, etc.) are supporteed by specifying the `type` subplot option in the `specs` argument to `make_subplots`.
+By default, the `make_subplots()` function assumes that the traces that will be added to all subplots are 2-dimensional cartesian traces (e.g. `scatter`, `bar`, `histogram`, `violin`, etc.).  Traces with other subplot types (e.g. `scatterpolar`, `scattergeo`, `parcoords`, etc.) are supporteed by specifying the `type` subplot option in the `specs` argument to `make_subplots`.
```

Comment:
in general, I don't really agree with putting the `()` after the function name everywhere (this comment applies to this whole PR btw). The function is properly named `make_subplots` and calling it with no parameters like `make_subplots()` will probably result in an error. It's probably OK to use this for something like "the `.show()` method" as that actually will work with no arguments.

Also note: maybe we haven't used it super consistently, but the leading `.` in "the `.show()` method" is actually informative IMO in that it reminds the reader that `show` isn't a free function but a method of a figure. As a matter of style we could say "the `fig.show` method" or "the `.show` method of every `go.Figure` object" or something.

In any case, unless you're going to fully standardize the usage of this kind of thing throughout the docs, I find it pretty distracting and hard to review changes like this :)